tests/routing/unit/ActiveLink.ts

```diff
@@ -244,4 +249,58 @@ describe('ActiveLink', () => {
 
 		assert.deepEqual(dNode2.properties.classes, []);
 	});
+
+	jsdomDescribe('integration tests', () => {
```

Comment:
should this be `describe`, not `jsdomDescribe`? Perhaps I'm misunderstanding, but I thought `jsdomDescribe` was the type here?

Reply:
Nope, this is the plugin to use jsdom.